samples/02.branding-styling-and-customization/j.activity-grouping/comprehensive.html

```diff
@@ -0,0 +1,650 @@
+<!DOCTYPE html>
```

Comment:
What's this file for? Do you need to add the link to the README.md? There should also be an explanation of what this file is.

Reply:
I also feel like this file is complicated enough that it should be npm, not CDN

Reply:
This is linked to `samples/README.md` to show a sample that allow the user to play around more comprehensively, rather than the need to tweak the code to get the result.

We have this comprehensive sample up for other sample, e.g. Bluetooth audio sample.

Reply:
Is it in the readme though?

Reply:
Resolved